index.php

```diff
@@ -45,6 +45,21 @@
         <li><a href="https://github.com/kaincenteno/wiki">wiki Github</a>
       </ul>
 
+    <h4>FOUND A BUG IN THE GAME</h4>
+      <p>If you have found a bug in the game. We encourage you to submit it through a discussion.
+      </p>
+      <ul>
+        <li><a href="https://github.com/LandSandBoat/server/discussionsi">LandSandBoat Server Github Discussion</a></li>
+      </ul>
+
+    <h4>RUN YOUR OWN SERVER</h4>
+      <p>You can find all the information needed to run your own private server by going to the projects
+        github wiki. We (xiweb) do not provide support for setting up your own server.
+      </p>
+      <ul>
+        <li><a href="https://github.com/LandSandBoat/server/wiki">LandSandBoat Server Github Wiki</a></li>
+      </ul>
+
     <h4>CONTRIBUTE UPSTREAM(LANDSANDBOAT REPO)</h4>
```

Comment:
Punctuation, missing a space between UPSTREAM and (LANDSANDBOAT, change to:
<h4>CONTRIBUTE UPSTREAM (LANDSANDBOAT REPO)</h4>